modules/specialized-guides/pages/salt/salt-ssh.adoc

```diff
@@ -27,8 +27,15 @@ Use this method for in-place firewall environments that block HTTP or HTTPS conn
 As with all Salt calls, {productname} invokes `salt-ssh` via the `salt-api`.
 
 Salt SSH relies on a roster to obtain details such as hostname, ports, and the SSH parameters of a client.
-{productname} keeps these details in the database and makes them available to Salt by generating a temporary roster file for each Salt SSH call.
-The location of the temporary roster file is supplied to `salt-ssh` using the `--roster-file= option`.
+{productname} keeps these details in the database and makes them available to Salt by using `uyuni` roster module or by generating a temporary roster file on bootstrapping new clients with {webui} using Salt SSH call.
+The location of the temporary roster file is supplied to `salt-ssh` using the `--roster-file=` option.
```

Comment:
ok, your explanation make sense (of course!).  In the long run, we should write "Salt SSH" when ever the subsystem is meant.  But this can come later, separately.  I think we now can merge your PR.  I unfortunately forgot again whether you have merge rights or not.  I can merge it for you.